### PR TITLE
new icon: matlab (original, plain)

### PR DESCRIPTION
**Pull request by Panquesito7**
I added the icon for the [MATLAB](https://en.wikipedia.org/wiki/MATLAB) programming language.

As I didn't find out any original images in `.svg`, I had to take it from the first commit in the
[`vscode-icons`](https://github.com/vscode-icons/vscode-icons/pull/2263) repository. I resized it using the [`resize-image`](https://www.iloveimg.com/resize-image/resize-svg) tool. 🙂
**I only need help setting up the color in `devicon.json`. I don't know how does it work exactly.**

This is how it looks:

MATLAB Icon MATLAB-Plain Icon

If I missed up anything please let me know. Thank you.

### CODE REVIEW

**Review by amacado — CHANGES_REQUESTED** [review]
Hello @Panquesito7 and thanks for adding matlab! :) Sadly the icon you provided does not fulfill our requirements.

As you can see in the screenshots posted by our bot the font version looks does not look like you might expect it. This is way we have introduct a `plain` version which simplifies a icon so it looks good in one-colored. Feel free to commit a plain version of matlab :) Since our guidelines require a valid font version of a icon I'm unable to approve your pull request :(

Side note: When I google the Matlab logo it looks more like a green-blue color (https://de.wikipedia.org/wiki/Matlab#/media/Datei:Matlab_Logo.png) than a solid blue like in your contribution...

**Comment by Panquesito7** [comment]
Oh sorry, I'm still kinda new to this thing. I wasn't sure if to keep the `font` thingy in `devicon.json`. Should I remove it?
And about the color...I tried searching for an official icon in `.svg` format. I guess I have to convert it manually or search somewhere else.

**Comment by Panquesito7** [comment]
About the color: Is this icon OK? https://icons8.com/icons/set/matlab

**Comment by Panquesito7** [comment]
Oops, the icon has a white background, working on fixing it.

**Comment by Panquesito7** [comment]
> Side note: When I google the Matlab logo it looks more like a green-blue color (https://de.wikipedia.org/wiki/Matlab#/media/Datei:Matlab_Logo.png) than a solid blue like in your contribution...

If you go to their official website, it is blue, not green-blue.

**Comment by Panquesito7** [comment]
Edited the first post.

**Comment by Panquesito7** [comment]
@amacado all issues have been fixed. Please see my previous comment, and please review my PR. Thank you. 🙂

File: devicon.json [hunk 1]
```diff
@@ -2052,6 +2052,24 @@
...
+        "versions": {
+          "svg": [
+            "original",
+            "plain"
+          ],
+          "font": [
+            "plain"
+          ]
+        },
+        "color": "#CC3333",
```
Comment (amacado) [inline devicon.json:2070]:
```suggestion
        "color": "#6dd0c7",
```

**Review by amacado — CHANGES_REQUESTED** [review]
I have no idea why the bot is reporting a error with `bash-plain.svg` and `devicon.svg`..  Strange behaviour.. I checked the icon and it looks good! Thanks @Panquesito7. I would only suggest to change to color (see my review suggestion)

**Comment by Panquesito7** [comment]
Done! Color changed.

**Review by amacado — APPROVED** [review]
I'm approving this now! It was a very confusing pull request due to the fact it was opened against the master branch on not develop branch..  Next time please open the pull request to `develop` branch (and start by checking out the `develop` branch) ;-)
Anyway: icon looking fine now! Thanks for adding matlab to this project! :) It will be included in the next release!

**Comment by Panquesito7** [comment]
> Next time please open the pull request to `develop` branch (and start by checking out the `develop` branch) ;-)

So sorry for this; it was my first time contributing, and I didn't do things well. Next time, I'll do everything as per the contributing guidelines. Thank you! 😄

**Pull request merged**